UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -351,63 +249,11 @@ static void OnQuitCommandReceived()
             Application.Quit();
         }
 
-        void OnResetCommand(EnvironmentResetParameters newResetParameters)
+        void OnResetCommand()
         {
-            UpdateResetParameters(newResetParameters);
             ForcedFullReset();
         }
 
-        void OnRLInputReceived(UnityRLInputParameters inputParams)
-        {
-            m_IsInference = !inputParams.isTraining;
-        }
-
-        void UpdateResetParameters(EnvironmentResetParameters newResetParameters)
-        {
-            if (newResetParameters.resetParameters != null)
-            {
-                foreach (var kv in newResetParameters.resetParameters)
-                {
-                    resetParameters[kv.Key] = kv.Value;
-                }
-            }
-            customResetParameters = newResetParameters.customResetParameters;
-        }
-
-        /// <summary>
-        /// Configures the environment settings depending on the training/inference
-        /// mode and the corresponding parameters passed in the Editor.
-        /// </summary>
-        void ConfigureEnvironment()
-        {
-            if (m_IsInference)
-            {
-                ConfigureEnvironmentHelper(m_InferenceConfiguration);
-                Monitor.SetActive(true);
```

Comment:
I've never used the Monitor so I don't know much about it, but these were the only places we're calling SetActive on it. Do these calls need to be moved somewhere?

Reply:
The monitor is on by default now, regardless of wether we are training or not.